tests/api/nb_ngram_api/test_make_word_graph.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import unittest.mock
+from dhlab.api.nb_ngram_api import make_word_graph
+import json
+import networkx as nx
+from typing import Literal
+
+
+def _links_to_edge_view(nodes: list[str], links: list[dict[str, int]]) -> dict[tuple[str, str], dict[Literal["value", int]]]:
+    """Converts nodes and links as returned by the API to a dictionary with the same structure as nx.OutEdgeView
+
+    Example
+    -------
+    >>> _links_to_edge_view(["a", "b", "c"], links=[{"source": 0, "target": 2, "value": 3}])
+    {('a', 'c'): {'weight': 3}}
+    """
+    return {
+        (nodes[link["source"]]["name"], nodes[link["target"]]["name"]): {"weight": link["value"]}
+        for link in links
+    }
+
+def test_deserialises_di_graph() -> None:
+    """The word graph is deserialised correctly
+    """
+    data = {
+        "nodes": [
+            {"name": "askeladden", "size": 4},
+            {"name": "de", "size": 33},
+            {"name": "Rødrev", "size": 24},
+            {"name": "prinsessen", "size": 33},
+        ],
+        "links": [
+            {"source": 0, "target": 1, "value": 66},
+            {"source": 2, "target": 0, "value": 9},
+            {"source": 3, "target": 0, "value": 8},
+        ],
+    }
+
+    session = unittest.mock.MagicMock(spec=requests.Session)
+    session.get.return_value.status_code = 200
+    session.get.return_value.text = json.dumps(data)
+
+    word_graph = make_word_graph(words=["mocked", "data"], session=session)
```

Comment:
```suggestion
    word_graph = make_word_graph(words="mocked,data", session=session)
```

Probably inconsequential when we're mocking the return-values, but make_word_graph expects this format.